src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -153,9 +158,10 @@ def _initialise(self):
         # Initialise the aperture motor. This also has the side effect of fully opening the iris.
         self._initialise_aperture()
 
-        # Initalise focus. First move the focus to the close stop.
-        self._move_zero()
-        self._min_position = 0
+        # Calibrate near stop of the astromechanics focuser.
+        self._calibrate_near_stop()
+
+        self._min_position = self._calibration_position
```

Comment:
I would change the name to `_zero_position` for consistency.

```suggestion
        # Calibrate near stop of the astromechanics focuser.
        self._move_zero()
        self._min_position = self._zero_position
```